scripts/ciauthenticate

```diff
@@ -15,5 +15,4 @@ CI authentication for this project.
 
 # curl -sL https://aka.ms/InstallAzureCLIDeb | bash
 # az login --service-principal --username ${CLIENT_ID} --tenant "microsoft.onmicrosoft.com" --password ${CLIENT_SECRET}
-az acr login --name pccomponentstest
-docker login pccomponentstest.azurecr.io --username ${CLIENT_ID} --password ${CLIENT_SECRET}
\ No newline at end of file
+az acr login --name pccomponentstest
```

Comment:
Curious if @ghidalgo3 can comment on why the acr and docker logins were both required, and if that would still hold even when using this new credential style.

Reply:
In #192, I had to add both `docker login` on top of the existing `az acr login` to allow the [ORAS](https://oras.land/) to authenticate against our ACRs. We still need to do the same under the new credential style.

Reply:
I think that once you `az login`, if the SP has the right permissions it can get the password for the ACR with these commands: https://learn.microsoft.com/en-us/cli/azure/acr/credential?view=azure-cli-latest

Reply:
I tested the full execution without `docker login` and authentication seems fine. Can we drop it? @ghidalgo3

Reply:
Yes, curious that `oras` took the az credentials when it wouldn't before... if it works it works!